frontend/src/libs/components/modal/steps/delete-account/delete-account-message.tsx

```diff
@@ -0,0 +1,29 @@
+import { type FC } from 'react';
+
+import { Button } from '#libs/components/components.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  onNext?: () => void;
+  onClose?: () => void;
+};
+
+const DeleteAccountMessage: FC<Properties> = ({ onNext, onClose }) => {
+  return (
+    <div className={styles['modal-body']}>
+      <p>After deleting your account you will:</p>
+      <ul className={styles['text']}>
+        <li>Not be able to login to CalmPal</li>
+        <li>Lose acces to your meditations</li>
+        <li>Your data and profile history will be deleted</li>
+      </ul>
+      <div className={styles['footer']}>
+        {onClose && <Button label="Cancel" style="primary" onClick={onClose} />}
+        {onNext && <Button label="Continue" style="primary" onClick={onNext} />}
```

Comment:
May we create separate boolean like `hasNextStep`?

Reply:
I think is not necessary, steps are managed by an array and the logic for advancing steps is handled by the goToNextStep function.

Reply:
Please follow @Dmytro-Kucherenko comment

Reply:
at first, I didn't understand correctly what you meant by the add the hasNextStep @Dmytro-Kucherenko , now i get it, thanks.